modules/gdscript/language_server/gdscript_language_protocol.cpp

```diff
@@ -345,4 +351,209 @@ GDScriptLanguageProtocol::GDScriptLanguageProtocol() {
 	set_scope("completionItem", text_document.ptr());
 	set_scope("workspace", workspace.ptr());
 	workspace->root = ProjectSettings::get_singleton()->get_resource_path();
+	scene_cache.workspace = workspace;
+}
+
+void SceneCache::_get_owners(EditorFileSystemDirectory *p_efsd, const String &p_path, List<String> &r_owners) {
```

Comment:
```suggestion
void SceneCache::_get_owners(EditorFileSystemDirectory *p_efsd, const String &p_path, LocalVector<String> &r_owners) {
```
`List` should not be used in new Code anymore as per [the c++ usage guidelines](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html)